### ISSUE TITLE
universal beta: default to nojson in package.json

### ISSUE THREAD

**Issue by dbachelder** [rollbar/rollbar.js#273]
In the universal beta the default include is `rollbar.umd.min.js`  when you `import Rollbar from 'rollbar'`

package.json:
```
  "browser": "dist/rollbar.umd.min.js",
```

which adds around 47k to a gzipped webpack app (our largest dependency). We alias it to `rollbar.umd.nojson.min.js` in the webpack config like this:

```
  resolve: {
    .... 
    alias: {
      "rollbar": path.resolve(__dirname, "node_modules/rollbar/dist/rollbar.umd.nojson.min.js")
    },
  },
```

to get it down to a bit over 9k. Would be nice if the smaller version was the default... although I'm not entirely sure I understand the distinction between these two distributions and I can't seem to find any docs.

**Comment by rokob** [comment]
The nojson ones shouldn't be there anymore. Let me investigate the size increase, but the stuff with nojson I just forgot to delete.

**Comment by dbachelder** [comment]
yeah.. I just noticed it's not working with that dependency anymore anyway.. I was so focused on getting the size down that I didn't bother testing the functionality :/

**Comment by rokob** [comment]
Okay I didn't remove the nojson files yet just so I could show this size comparison on github. I did some work to get the size of the working umd.min.js build down:

https://github.com/rollbar/rollbar.js/blob/4b04ff960e5e15d2a1647426f8c2282d6fae9461/dist/rollbar.umd.min.js
38.1 KB

vs.

https://github.com/rollbar/rollbar.js/blob/4b04ff960e5e15d2a1647426f8c2282d6fae9461/dist/rollbar.umd.nojson.min.js
25.4 KB

Note that the umd.nojson.min.js is just old code so it wouldn't be working anyway, but this gets them in a similar ballpark.

Gzip'd the nojson one comes in around 9.4KB and the working one comes in around 12.8K. Hopefully this extra 3K isn't the end of the world. I can try to improve size more, but I think I got the biggest chunk I could getting from 164KB down to 38KB uncompressed.
